class_teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'class_person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, name:, parent_permission:, specialization:)
+    super(age, name: name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] I see you are defaulting the argument in the super, when you pass arguments to super, you're providing values for the parameters of the superclass's constructor, so the value should be inside the initialize. Apply this to the student class too, please.

Reply:
@Olamarx Thanks for your review, I'll check it out.